Source/SwiftLintFramework/Rules/Lint/PrivateUnitTestRule.swift

```diff
@@ -63,6 +63,11 @@ public struct PrivateUnitTestRule: ASTRule, ConfigurationProviderRule, CacheDesc
                 "internal func test2() {}\n " +
                 "public func test3() {}\n " +
             "}",
+            "@objc private class FooTest: XCTestCase { " +
```

Comment:
In the future, you can use multiline string literals and format this as you would normal Swift code. The code layout here is a bit odd. No big deal, we can still merge with this.